concent_api/core/tests/utils.py

```diff
@@ -204,6 +205,16 @@ def _test_database_objects(
             self.assertEqual(ReceiveStatus.objects.last().delivered,        receive_delivered_status)
             self.assertEqual(ReceiveStatus.objects.last().message.task_id,  task_id)
 
+    def _test_subtask_state(self, task_id, subtask_id, subtask_state, provider_key, requestor_key, next_deadline = None):
+        last_subtask = Subtask.objects.last()
```

Comment:
It will not always be last() Subtask, so please get it by `subtask_id`.

Reply:
:+1: